src/main/service/listManipulation/CivilFamilyAndMixedListService.ts

```diff
@@ -15,11 +15,39 @@ export class CivilFamilyAndMixedListService {
      * Manipulate the family/mixed json data for rendering to the screen.
      * @param list
      */
-    public sculptedFamilyMixedListData(list: string): object {
+    public sculptedFamilyMixedListData(list: string, partyAtHearingLevel = false): object {
```

Comment:
I'm wondering if this method adds value. A similar if statement is done in the controller. Could the controller just call the sculpted list data methods this one calls directly?

Reply:
Have removed the top level methods so the controller calls them directly.